recipes/thriftpy/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py build_ext
```

Comment:
Doesn't this happen automatically when installing? In any event, not a big deal either way.